sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/annotations/DefaultSchema.java

```diff
@@ -97,7 +100,7 @@ private SchemaProvider getSchemaProvider(TypeDescriptor<?> typeDescriptor) {
                       + " specified as the default SchemaProvider for type "
                       + type
```

Comment:
This is a TypeDescriptor which doesn't have a getName. Are you suggesting type.getRawType().getName()?

Reply:
Sorry, my bad, I was referring to the code that uses `Class` not  `TypeDescriptor`. This code is good because `TypeDescriptor#toString` is equivalent to `type.getRawType().getName()`